packages/experimental-utils/package.json

```diff
@@ -33,8 +33,8 @@
     "postbuild": "downlevel-dts dist _ts3.4/dist",
     "clean": "tsc -b tsconfig.build.json --clean",
     "postclean": "rimraf dist && rimraf _ts3.4 && rimraf coverage",
-    "format": "prettier --write \"./**/*.{ts,js,json,md}\" --ignore-path ../../.prettierignore",
-    "lint": "eslint . --ext .js,.ts --ignore-path='../../.eslintignore'",
+    "format": "prettier --write \"./**/*.{ts,mts,cts,tsx,js,mjs,cjs,jsx,json,md,css}\" --ignore-path ../../.prettierignore",
```

Comment:
Unless I'm missing something, we can use `prettier --write \"./**/*\" --ignore-unknown --ignore-path ../../.prettierignore`?

Reply:
I think we did this to control the files that get formatted... I forget exactly - it's been so long!
So I just kept it as is. I don't know if we really need to bother with this now.